sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -82,7 +81,7 @@ def _get_node_load(self) -> tuple[float, float, float]:
         except Exception as exc:  # pylint: disable=broad-except
             LOGGER.debug("Couldn't get node load from prometheus metrics. Error: %s", exc)
             # fallback to uptime
-            load_1, load_5, load_15 = self.remoter.run('uptime', retry=3).stdout.split("load average: ")[1].split(",")
+            load_1, load_5, load_15 = self.remoter.run('uptime').stdout.split("load average: ")[1].split(",")
```

Comment:
Removal of retries also sounds as a separate fix which requires description.
It is not clear now why it is needed at all. How exactly it makes thing be better?